osquery/tests/integration/tables/known_hosts.cpp

```diff
@@ -16,25 +16,16 @@
 
 namespace osquery {
 
-class knownHosts : public IntegrationTableTest {};
+class KnownHostsTest : public IntegrationTableTest {};
 
-TEST_F(knownHosts, test_sanity) {
-  // 1. Query data
-  // QueryData data = execute_query("select * from known_hosts");
-  // 2. Check size before validation
-  // ASSERT_GE(data.size(), 0ul);
-  // ASSERT_EQ(data.size(), 1ul);
-  // ASSERT_EQ(data.size(), 0ul);
-  // 3. Build validation map
-  // See IntegrationTableTest.cpp for avaialbe flags
-  // Or use custom DataCheck object
-  // ValidatatioMap row_map = {
-  //      {"uid", IntType}
-  //      {"key", NormalType}
-  //      {"key_file", NormalType}
-  //}
-  // 4. Perform validation
-  // validate_rows(data, row_map);
+TEST_F(KnownHostsTest, sanity) {
+  QueryData rows = execute_query("select * from known_hosts");
+  ValidatatioMap row_map = {
+      {"uid", IntType},
+      {"key", NonEmptyString},
+      {"key_file", FileOnDisk},
+  };
+  validate_rows(rows, row_map);
```

Comment:
Check if rows is empty and if so LOG(WARNING) that table won't be actually tested.

Reply:
Maybe we could even put that under `validate_rows()`.